src/main.cpp

```diff
@@ -875,3 +877,137 @@ int main(int argc, char *argv[])
 
   return 0;
 }
+
+#else
+
+#if !defined(NODE_MAX)
```

Comment:
I think i'd prefer a special fuzz only binary instead of doing ifdefs everywhere. That way you can just disable BTF support completely instead of what happens now. 

So in `src/CMakelist.txt`

```
add_executable(bpftrace_fuzz, ..files.. fuzz_main.cpp ...)
```

Reply:
I think we also want to fuzz BTF related part, but the separate a binary seems good to me. I'll try it.

Reply:
I added fuzz_main.cpp, and now when building fuzz binary that name is bpftrace_fuzz. I'll rebase commits if it's desirable.